examples/sandbox/package.json

```diff
@@ -21,7 +21,7 @@
     "@shapeshiftoss/hdwallet-ledger": "1.55.9",
```

Comment:
sandbox should ref 1.55.11 too

Reply:
<img width="714" alt="image" src="https://github.com/user-attachments/assets/9c54e829-2ca2-43bc-a00f-83ad3911aabe">

Doesn't look like we're able to do it in the same PR and are inherently one version ahead?

Reply:
we removed coinbase so that is why it isn't able to ref a local package

Reply:
🧠 de2247e6